presto-main/src/main/java/com/facebook/presto/sql/relational/SqlFunctionUtils.java

```diff
@@ -191,30 +187,6 @@ public Expression rewriteIdentifier(Identifier node, Map<Identifier, VariableRef
         }, sqlFunction, variableMap);
     }
 
-    private static Expression coerceIfNecessary(FunctionMetadata functionMetadata, Expression sqlFunction, SqlFunctionProperties sqlFunctionProperties, Metadata metadata)
```

Comment:
Let's keep this. We need to check all existing functions and update them before it's safe to remove these. So also add the coercion in `getSqlFunctionExpression`. We can think about introducing a warning for this and deprecate this in later releases.

Reply:
Ok sure, done.